src/scripts/mutual_checker.js

```diff
@@ -19,12 +19,15 @@ let primaryBlogName;
 let postAttributionSelector;
 let icon;
 
+const needsReprocessing = postElement =>
+  postElement.classList.contains(mutualsClass) &&
+  postElement.querySelector(`.${mutualIconClass}`) === null;
+
 const addIcons = function (postElements) {
-  [...document.querySelectorAll(`${postSelector}.${mutualsClass}`)]
-    .filter(postElement => postElement.querySelector(`.${mutualIconClass}`) === null)
-    .forEach(postElement => postElement.classList.remove(excludeClass));
+  filterPostElements(postElements, { includeFiltered: true }).forEach(async postElement => {
+    if (postElement.classList.contains(excludeClass) && !needsReprocessing(postElement)) return;
+    postElement.classList.add(excludeClass);
```

Comment:
I feel like we could remove `excludeClass` altogether here and simplify the condition down to

```suggestion
    if (needsReprocessing(postElement) === false) return;
```

since this by itself should prevent double mutual icons from appearing.

Reply:
Er... oh. Oh, because this only fires on new postElements. Right. Because we implemented a mutation observer almost a month ago.

That took me way, way too long to figure out :P

Reply:
Er, wait, no, this only processes posts that need to be reprocessed, right?